packages/core/styled-system/src/config/background.ts

```diff
@@ -5,7 +5,7 @@ import { t, Token, transforms } from "../utils"
 export const background: Config = {
   background: t.colors("background"),
   backgroundColor: t.colors("backgroundColor"),
-  backgroundImage: t.propT("backgroundImage", transforms.bgImage),
```

Comment:
This one is a breaking change. I was hoping we could add this while preserving the existing transforms for `bgGradient`

Reply:
@segunadebayo I ended up moving it's logic over to the `parseGradient` function, hence the tests I added still passing. Will this still mot work?

Reply:
@segunadebayo I moved it's logic over to the `parseGradient` function, hence the test I added passed with the `url()`. Is there a check I am missing?

Reply:
I'm just now recognizing that I should be adding tests to `parse-gradient.test.ts` instead. But it appears there are no tests failing there at the moment.

Reply:
Got it. I'm going to double check that.